src/components/Menu.jsx

```diff
@@ -1,11 +1,29 @@
-import React from 'react'
+import React, { useState, useEffect } from "react";
+import { Segment, List } from "semantic-ui-react";
+import MenuItem from "./MenuItem";
+import axios from "axios";
 
 const Menu = () => {
+  const [menuItems, setIdentification] = useState([]);
+  useEffect(() => {
+    axios.get("https://slowfood.heroku.com/api/products").then((response) => {
+      setIdentification(response.data.products);
+    });
+  }, []);
+
+  let menuList = menuItems.map((items) => {
```

Comment:
When you iterate over this array, wouldn't it be better to use the name `item` vs `items`? And then consequently change it below when you pass it into the `MenuItem` component.